scripts/app.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
```

Comment:
This is not really a script. It is the application.  Move the file to the project rood folder.